packages/nx/src/command-line/release/version/release-group-processor.ts

```diff
@@ -65,6 +72,7 @@ export const BUMP_TYPE_REASON_TEXT = {
     ', because a dependency project belonging to another release group was bumped, ',
   OTHER_PROJECT_IN_FIXED_GROUP_WAS_BUMPED_DUE_TO_DEPENDENCY:
     ', because of a dependency-only bump to another project in the same fixed release group, ',
+  NOOP_VERSION_ACTIONS: ', because this project uses docker, ',
```

Comment:
Should this explain that it uses docker _and_ that the docker config has caused it to skip version actions? (More succinctly than that obviously)